examples/hello_pytorch/pixi.toml

```diff
@@ -0,0 +1,30 @@
+[project]
+authors = ["Matthew Feickert <matthew.feickert@cern.ch>"]
+channels = ["conda-forge"]
+description = "Add a short description here"
+name = "hello_pytorch"
+platforms = ["linux-64"]
+version = "0.1.0"
+
+[tasks]
+# c.f. https://gist.github.com/matthewfeickert/a617c83a1bc3b4b2fd33b06bd267de76
+detect-gpu = """
+python ./src/torch_detect_GPU.py
+"""
+
+mnist-test = """
+python ./src/torch_MNIST.py
+"""
+
+start = { depends-on = ["mnist-test"] }
+
+[system-requirements]
+cuda = "11"
```

Comment:
`pixi` team: Is there a way for me to still specify what version of `cuda` that I need so that `pixi` knows that the version of `pytorch-gpu` that I need is the `cuda11` version but such that I don't _need_ to get all of the CUDA binaries that aren't already installed in the `ghcr.io/prefix-dev/pixi:jammy-cuda-11.8.0` base image?

Though, it seems that I'm doing this wrong as 

```console
$ grep 'pytorch-gpu-2.5.1' pixi.lock
      - conda: https://conda.anaconda.org/conda-forge/linux-64/pytorch-gpu-2.5.1-cuda126hf7c78f0_303.conda
      - conda: https://conda.anaconda.org/conda-forge/linux-64/pytorch-gpu-2.5.1-cuda126hf7c78f0_303.conda
- conda: https://conda.anaconda.org/conda-forge/linux-64/pytorch-gpu-2.5.1-cuda126hf7c78f0_303.conda
```

is showing that I'm getting the `cuda12` versions when I want the `cuda11`.